nbs/models.deepnpts.ipynb

```diff
@@ -0,0 +1,373 @@
+{

```

Comment:
<p>Why is this not being run? We should check that it runs successfully to avoid breaking it in the future, if it takes too long just set max_steps=1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Ah, yes. So I started this model with a copy from `DeepAR`, where this is also included, and forgot to remove it in the end. I'll remove it, this model is fast, so shouldn't give an issue in terms of time.